src/RequestMetadata.php

```diff
@@ -4,29 +4,60 @@
 
 namespace IronCore;
 
+/**
+ * Holds metadata fields as part of an encrypted document. Each encrypted document will have
+ * metadata that associates it to a tenant ID, which service is accessing the data,
```

Comment:
I feel like this is difficult to read. Maybe "Each document has metadata associated with it. Some examples include..."

Reply:
I mostly copied this from the Java SDK, but I agree. I tried re-wording it, so please take another look.